app/deliver_grant_funding/forms.py

```diff
@@ -195,3 +203,17 @@ class QuestionForm(FlaskForm):
         widget=GovTextInput(),
     )
     submit = SubmitField(widget=GovSubmitInput())
+
+
+class ShareGrantUserForm(FlaskForm):
+    user_email = StringField(
+        "This needs to be the user’s personal 'communities.gov.uk' email address, not a shared email address.",
+        validators=[
+            DataRequired("Enter the email address"),
+            Email(message="Enter an email address in the correct format, like name@communities.gov.uk"),
```

Comment:
We have a few uses of the Email validator now with duplicative error messaging. This doesn't match the other ones, albeit presumably on purpose to include the communities domain.

Can we create a reusable Email validator that can either manage generic emails or communities emails, and then just has the error message content defined in that one place? I see four other uses of the Email validator that would need updating.